src/tap/README.md

```diff
@@ -77,6 +77,95 @@ cut_rows [-endcap_master endcap_master]
 - `-halo_width_x`. Specify the horizontal halo size (in microns) around macros during cut rows.
 - `-halo_width_y`. Specify the vertical halo size (in microns) around macros during cut rows.
 
+### Only adding boundary cells
+
+```
+place_boundary_cells
```

Comment:
Could these options be derived from LEF58_CLASS rather than requiring the user to specify them?  The list in LEF is even longer.

Can we use the same naming scheme as LEF if we keep the options?

In LEF these all fall under ENDCAP types.  What do you think of place_endcaps instead of place_boundary_cells?

Reply:
I'll check the naming for the LEF_58_TYPES, I would be fine with that if its present on all cells.
`place_endcaps` would be fine as well.

Reply:
I may also be able to simplify some of the R0/MX logic